news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "The Followpoint: BTMC, the Face of osu!"
```

Comment:
```suggestion
title: "The Followpoint: BTMC, the face of osu!"
```

Reply:
Not sure if uppercase is suppose to be there

Reply:
it's in title case, so yes